.github/workflows/python-package.yml

```diff
@@ -42,15 +42,40 @@ jobs:
         python -m pip install --upgrade pip
         python -m pip install flake8 pytest
         if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
+    - name: Test with pytest
+      run: pytest --benchmark-skip
```

Comment:
What does this end up doing if it skips benchmarks? I guess it runs other tests unit//integration?

Reply:
Indeed. Basically, we want to run all the other non-perf test first, and then only run perf test when the MSAL is functionally correct. Otherwise there will be no point to benchmark an incorrect commit/PR.